src/SME.SGP.Dados/Repositorios/RepositorioEncaminhamentoAEE.cs

```diff
@@ -150,9 +150,10 @@ public async Task<EncaminhamentoAEE> ObterEncaminhamentoPorId(long id)
 
         public async Task<EncaminhamentoAEE> ObterEncaminhamentoComTurmaPorId(long encaminhamentoId)
         {
-            var query = @"select ea.*, t.*
+            var query = @" select ea.*, u.nome as ResponsavelNome, U.rf_codigo as ResponsavelRf, t.*
```

Comment:
Carregar a entidade para buscar os campos depois
```suggestion
            var query = @" select ea.*, u.*, t.*
```